backends/jgroups/src/test/resources/log4j.properties

```diff
@@ -24,10 +18,10 @@ log4j.logger.org.jboss=info
 
 log4j.logger.org.hibernate=warn
 
-log4j.logger.org.hibernate.search=debug
+log4j.logger.org.hibernate.search=info
+log4j.logger.org.hibernate.search.backend=info
```

Comment:
Is there a specific reason to do that? I'd rather have extensive logs in tests by default... It's particularly useful when encountering transient failures on CI. We already use the `debug`/`trace` level in Elasticsearch integration tests, for instance.

Reply:
ok I can revert these.
I typically prefer to "opt-in" on the details as there was too much noise but you have a good point about CI failures.